honeybee_grasshopper_energy/src/HB Ventilation.py

```diff
@@ -0,0 +1,79 @@
+            that ventilation is varied based on real-time occupancy but rather
+            that the design level of ventilation is determined using this value
+            and the People object of the zone. To vary ventilation in real time,
+            the ventilation schedule should be used. Most ventilation standards
+            support that a value of 0.01 m3/s (10 L/s or ~20 cfm) per person is
+            sufficient to remove odors. Accordingly, setting this value to 0.01
+            and using 0 for the following ventilation terms will often be suitable
+            for many applications. Default: 0.
+        _flow_per_area_: A numerical value for the intensity of ventilation in m3/s
+            per square meter of floor area. Default: 0.
+        _ach_: A numberical value for the design level of ventilation
+            in air changes per hour (ACH) for the entire zone. This is particularly
+            helpful for hospitals, where ventilation standards are often given
+            in ACH. Default: 0.
+        _schedule_: An optional fractional schedule for the ventilation over the
+            course of the year. The fractional values will get multiplied by
+            the total design flow rate (determined from the fields above and the
+            calculation_method) to yield a complete ventilation profile. Setting
+            this schedule to be the occupancy schedule of the zone will mimic demand
+            controlled ventilation. If None, the design level of ventilation will
+            be used throughout all timesteps of the simulation. Default: None.
+    
+    Returns:
+        vent: An Ventilation object that can be used to create a ProgramType or
+            be assigned directly to a Room.
+"""
+
+ghenv.Component.Name = "HB Ventilation"
+ghenv.Component.NickName = 'Ventilation'
+ghenv.Component.Message = '0.1.0'
+ghenv.Component.Category = "Energy"
+ghenv.Component.SubCategory = '3 :: Loads'
+ghenv.Component.AdditionalHelpFromDocStrings = "1"
+
+import uuid
+
+try:
+    from honeybee_energy.load.ventilation import Ventilation
+    from honeybee_energy.lib.schedules import schedule_by_name
+    from ladybug_rhino.grasshopper import all_required_inputs
+except ImportError as e:
+    raise ImportError('\nFailed to import honeybee:\n\t{}'.format(e))
+
+
+# make a default Ventilation name if none is provided
+if _name_ is None:
+    _name_ = "Ventilation_{}".format(uuid.uuid4())
+
+# get the schedule
+if isinstance(_schedule_, str):
+    _schedule_ = schedule_by_name(_schedule_)
+
+# get default _flow_per_person_, _flow_per_area_, and _ach_
+_flow_per_person_ = _flow_per_person_ if _flow_per_person_ is not None else 0.0
+_flow_per_area_ = _flow_per_area_ if _flow_per_area_ is not None else 0.0
+_ach_ = _ach_ if _ach_ is not None else 0.0
+
+# create the Ventilation object
+vent = Ventilation(_name_, _flow_per_person_, _flow_per_area_,
+                    0, _ach_, _schedule_)
```

Comment:
Ok. It's `flow_per_zone`. Maybe I should just expose it on the component.  I originally didn't do it because OpenStudio standards gem doesn't use it but I can see that some people might want to use it.